redash/models.py

```diff
@@ -627,18 +627,25 @@ class QueryResult(db.Model, BelongsToOrgMixin):
     data = Column(db.Text)
     runtime = Column(postgresql.DOUBLE_PRECISION)
     retrieved_at = Column(db.DateTime(True))
+    data_scanned = Column(db.String(255), nullable=True)
 
     __tablename__ = 'query_results'
 
     def to_dict(self):
+        if hasattr(self, 'data_scanned') and self.data_scanned != '' and self.data_scanned is not None:
```

Comment:
I think Python's default bool checks would accomplish the same result? But if you prefer to be more explicit in what you're checking that's fine too.

`if hasattr(self, 'data_scanned') and self.data_scanned:`